src/contract.rs

```diff
@@ -9,24 +8,24 @@ use crate::state::{config, config_read, State};
 
 // Note, you can use StdResult in some functions where you do not
 // make use of the custom errors
-pub fn init<S: Storage, A: Api, Q: Querier>(
-    deps: &mut Extern<S, A, Q>,
+pub fn init(
+    deps: DepsMut,
     _env: Env,
     info: MessageInfo,
     msg: InitMsg,
-) -> StdResult<InitResponse> {
+) -> Result<InitResponse, ContractError> {
```

Comment:
This only returns `StdError`. No need to include `ContractError` here unless we use it explicitly.
(Not wrong either as `StdError` transparently converts to `ContractError`